src/dmd/libomf.d

```diff
@@ -167,29 +167,33 @@ final class LibOMF : Library
 
     /*****************************************************************************/
 
-    void addSymbol(OmfObjModule* om, const(char)* name, int pickAny = 0)
+    void addSymbol(OmfObjModule* om, const(char)[] name, int pickAny = 0)
     {
         static if (LOG)
         {
-            printf("LibOMF::addSymbol(%s, %s, %d)\n", om.name.ptr, name, pickAny);
+            printf("LibOMF::addSymbol(%s, %s, %d)\n",
+                cast(int)om.name.length, om.name.ptr,
+                cast(int)name.length, name.ptr, pickAny);
         }
-        const namelen = strlen(name);
-        StringValue* s = tab.insert(name, namelen, null);
+        StringValue* s = tab.insert(name, null);
         if (!s)
```

Comment:
It looks like there's a missed opportunity for code clarity using `if (sStringValue* s = ... ) {} else {}`. As far as I can see `s` is not used out of the block.